### ISSUE TITLE
Alerts - Have a When trigger that looks for when ownership Change to team X to send out slack alert to Team X

### ISSUE THREAD

**Issue by Angelodaniel** [getsentry/sentry#81456]
### Problem Statement

When Team A is the overseeing team responsible for some stack frames across multiple projects they want to be alerted in Slack when the project's teams assign the issues to Team A.

The re-assigning happens after the issue is being created and it should not fire everytime a event gets into an existing issue as that would create noise.

### Solution Brainstorm

When Ownership changes on Issue A and this ownership is Team A there needs to be an alert that fires to the Slack channel of Team A.
Everything is in place already except the "When ownership changes to Team X" trigger

### Product Area

Alerts

**Comment by ceorourke** [comment]
Team workflow notifications were built but never released, I'll put this in our backlog to look into.

**Comment by mltsy** [comment]
This would be very helpful for us!  Notifying "suggested assignees" on create is the best alternative I can find, but we often re-assign issues to other teams, and could really use this notification!

**Comment by Angelodaniel** [comment]
Hey Team, any updates on this? Still on the backlog?

**Comment by kcons** [comment]
Yes, this is still in the backlog, we're currently busy reworking/generalizing our alerting engine backend in order to make features like this easier to implement and maintain.
